fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
+
```

Comment:
> What are your thoughts on increasing its coverage. Shouldn't the basic scenario regarding running a very small python call with its module be enough?

You're just importing a module. There is no guarantee that the module is indeed working.
The best way we have to guarantee that is by running the test provided by the Python testsuite.

> You mean adding this line inside the IF which corresponds to the python version? Like inside elseif { [string match 3.* $pyver] } ?

That's one possible solution.  Just be careful to run it for each possible AT version that will support the sqlite module.